plugins/modules/zabbix_maintenance.py

```diff
@@ -44,11 +44,18 @@
             - Length of maintenance window in minutes.
         default: 10
         type: int
+    start_time:
+        description:
+            - Time the maintenance window starts in format YYYY-mm-dd HH:MM
+            - If None the default start_time is now()
+        default: None
+        type: str
     name:
         description:
             - Unique name of maintenance window.
-        required: true
+        required: false
         type: str
+        default: 'Created at now.strftime('%Y-%m-%d %H:%M')'
```

Comment:
Not sure what causes `ansible-doc -t module community.zabbix.zabbix_maintenance` fail, but I would guess that this may be a problem. Probably expects basic keywords like true/false or choices from other options.